src/modules/mc_pos_control/PositionControl/ControlMath.hpp

```diff
@@ -47,12 +47,18 @@ namespace ControlMath
 {
 /**
  * Converts thrust vector and yaw set-point to a desired attitude.
- * @param thr_sp a 3D vector
+ * @param thr_sp desired 3D thrust vector
  * @param yaw_sp the desired yaw
- * @return vehicle_attitude_setpoints_s structure
+ * @param att_sp attitude setpoint to fill
  */
-void thrustToAttitude(const matrix::Vector3f &thr_sp, const float yaw_sp,
-		      vehicle_attitude_setpoint_s &attitude_setpoint);
+void thrustToAttitude(const matrix::Vector3f &thr_sp, const float yaw_sp, vehicle_attitude_setpoint_s &att_sp);
+/**
+ * Converts a body z vector and yaw set-point to a desired attitude.
+ * @param body_z a world frame 3D vector in direction of the desired body z axis
+ * @param yaw_sp the desired yaw setpoint
+ * @param att_sp att_sp attitude setpoint to fill
```

Comment:
```suggestion
 * @param att_sp attitude setpoint to fill
```

Reply:
Fixed in place: https://github.com/PX4/Firmware/compare/08c397a3c528f80e4d26bcb9f6c1567fd2fdeae3..0d2e3463adf676661ebd46ca353df4d969506111